### PR TITLE
Milestone 4 - Add join table for visits

### PR DESCRIPTION
**Pull request by Ghiftee**
Here, I worked on the following:

- Create a vets table
- Create join tables called specializations and visits
- Insert data into created tables
- Query tables
 Please review my code
Thank you

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -71,17 +71,70 @@ SELECT species.name, COUNT(*) FROM animals
...
+    JOIN specializations ON specializations.vet_id = vets.id
+    WHERE specializations.species_id <> animals.species_id;
+
+SELECT species.name, COUNT(visits.animal_id) FROM visits
+    JOIN vets ON vets.id = visits.vet_id
+    JOIN animals ON animals.id = visits.animal_id
+    JOIN species ON species.id = animals.species_id
+    WHERE vets.name = 'Maisy Smith'
+    GROUP BY species.name
+    ORDER BY COUNT(visits.animal_id) DESC LIMIT 1;
```
Comment (DeliceLydia) [inline queries.sql]: - [x] Please add a query for this question `Who was the last animal seen by William Tatcher?` I can't find it

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Ghiftee, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Ghiftee) [reply queries.sql]: Thank you for the review. The query you're asking for is the first one however it returned the animal id, I have adjusted it to return the animal's name instead

File: queries.sql [hunk 2]
```diff
@@ -71,17 +71,72 @@ SELECT species.name, COUNT(*) FROM animals
...
 SELECT owners.full_name, COUNT(animals.owner_id) FROM animals 
     FULL OUTER JOIN owners 
     ON animals.owner_id = owners.id
-    GROUP BY owners.id;
\ No newline at end of file
+    GROUP BY owners.id;
+
+/* Add join table for visits */
+
+SELECT animals.name, visits.date_of_visit FROM visits
```
Comment (oxenprogrammer) [inline queries.sql:91]: Changes made :+1:

**Review by oxenprogrammer — APPROVED** [review]
Hi @Ghiftee , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**